docs/getting-started/first-routing.md

```diff
@@ -0,0 +1,200 @@
+---
+template: main.html
+---
+
+# Automated blue-green rollout
```

Comment:
I think our capitalization of titles and subtitles is inconsistent across all the docs. We should try to align everything at some point.